frontend/src/scenes/early-access-features/EarlyAccessFeatures.tsx

```diff
@@ -55,6 +70,17 @@ export function EarlyAccessFeatures(): JSX.Element {
                 }
                 delimited
             />
+            {showIntro && (
+                <ProductIntroduction
+                    productName="Early Access Features"
+                    productKey={ProductKey.EARLY_ACCESS_FEATURES}
+                    thingName="an early access feature"
```

Comment:
The other ones don't have `a/an` in front of it. Is this necessary? `Create early access feature` is shorter and just as understandable as `Create an early access feature` IMO

Reply:
Could just say "create feature"?